<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PbfShow</class>
 <widget class="QMainWindow" name="PbfShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>552</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>532</width>
    <height>552</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>532</width>
    <height>552</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PbfShow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnChoose">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>5</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>choose</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvCanvas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>452</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
